packages/holocron/src/ducks/constants.js

```diff
@@ -16,3 +16,9 @@ export const HOLOCRON_STORE_KEY = 'holocron';
 export const MODULES_STORE_KEY = 'modules';
 export const REDUCER_KEY = '@@holocron-module-reducer';
 export const LOAD_KEY = '@@holocron-module-load-action';
+export const REGISTER_MODULE_REDUCER = '@@holocron/REGISTER_MODULE_REDUCER';
+export const MODULE_LOADED = '@@holocron/MODULE_LOADED';
+export const MODULE_LOAD_FAILED = '@@holocron/MODULE_LOAD_FAILED';
+export const MODULE_LOADING = '@@holocron/MODULE_LOADING';
+export const MODULE_REDUCER_ADDED = '@@holocron/MODULE_REDUCER_ADDED';
+export const INIT_MODULE_STATE = '@@holocron/INIT_MODULE_STATE';
```

Comment:
The Constants scattered around the files of `holocron` is a code smell. I migrated the action constants to `ducks/constants.js` to standardize this.